proposals/simd/SIMD.md

```diff
@@ -666,6 +666,15 @@ natural alignment.
 
 Load a `v128` vector from the given heap address.
 
+Extended loads:
```

Comment:
Can this be made clear here that this is not operating on 64bit widths widening to a 128-bit vector? Having a concise description here will be useful.

Reply:
Do you mean to clarify that we are extending every lane, rather than widening a 64-bit value?